sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_constants.py

```diff
@@ -34,6 +34,8 @@ class SchemaFormat(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     AVRO = "Avro"
     """Represents the Apache Avro schema format."""
 
+    JSON = "JSON"
```

Comment:
TODO:
do we want to add Custom to this release?

TODO:
check with Anna/Johan, do we want "JSON" or "Json"? .NET is going with "JSON" as per the official capitalization.

Reply:
as per Anna: we can do "Json"